src/app/App.jsx

```diff
@@ -37,7 +37,7 @@ function App() {
       setUser("");
       return navigate("/login");
     },
-    staleTime: CONSTANT.oneHourInMillisecond,
+    staleTime: CONSTANT.ONE_HOUR_IN_MILLISECOND,
```

Comment:
정말 사소한 것이긴 하지만, Milliseconds (복수형)으로 변경할 수 있으면 좋겠습니다!

코드 곳곳에서 이 상수가 쓰이는 것 같은데, 수정된다면 같이 꼼꼼하게 수정해놔야겠네요 👍

```suggestion
    staleTime: CONSTANT.ONE_HOUR_IN_MILLISECONDS,
```

Reply:
지적 감사합니다! 반영할게요!